modules/vrtcalBidAdapter.js

```diff
@@ -0,0 +1,84 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
+import {ajax} from '../src/ajax';
+
+export const spec = {
+  code: 'vrtcal',
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid: function (bid) {
+    return true;// No extras params required
+  },
+  buildRequests: function (bidRequests) {
+    const requests = bidRequests.map(function (bid) {
+      const params = {
+
+        prebidJS: 1,
+        prebidAdUnitCode: bid.adUnitCode,
+        id: bid.bidId,
+        imp: [{
+          id: '1',
+          banner: {
+            w: bid.sizes[0][0],
+            h: bid.sizes[0][1],
```

Comment:
May want to look at the `mediaTypes.banner.sizes` field instead of the `bid.sizes` field, as that generic field may be deprecated in the future.